interface/super/rules/www/css/rules.css

```diff
@@ -157,3 +153,7 @@ root {
 input[type='text'].short {
   width: 80px;
 }
+
+.intervals span select {
+  width: inherit;
+}
```

Comment:
I chose to edit this file instead of the scss one because it felt a bit specific for this page, does it make sense?

Reply:
I'll have to check